### ISSUE TITLE
Trouble installing package

### ISSUE THREAD

**Issue by emarcotte** [mbarbon/perl-remote-debugging-client#2]
I'm having some trouble installing 0.14, seems like the makefile being generated is interpolating an array:

```
cpan install Devel::Debug::DBGp
CPAN: Storable loaded ok (v2.34)
Reading '/home/emarcotte/.cpan/Metadata'
  Database was generated on Mon, 01 Aug 2016 12:17:02 GMT
Running install for module 'Devel::Debug::DBGp'
Running make for M/MB/MBARBON/Devel-Debug-DBGp-0.14.tar.gz
CPAN: Digest::SHA loaded ok (v5.71)
CPAN: Compress::Zlib loaded ok (v2.069)
Checksum for /home/emarcotte/.cpan/sources/authors/id/M/MB/MBARBON/Devel-Debug-DBGp-0.14.tar.gz ok
CPAN: File::Temp loaded ok (v0.22)
CPAN: Parse::CPAN::Meta loaded ok (v1.4422)
CPAN: CPAN::Meta loaded ok (v2.150005)
CPAN: Module::CoreList loaded ok (v5.20160720)

  CPAN.pm: Building M/MB/MBARBON/Devel-Debug-DBGp-0.14.tar.gz

WARNING: OBJECT takes a string/number not a ARRAY reference.
         Please inform the author.
Checking if your kit is complete...
Looks good
Writing Makefile for Devel::Debug::DBGp
Writing MYMETA.yml and MYMETA.json
cp DB/DbgrURI.pm blib/lib/dbgp-helper/DB/DbgrURI.pm
cp DB/RedirectStdOutput.pm blib/lib/dbgp-helper/DB/RedirectStdOutput.pm
cp DB/OSType.pm blib/lib/dbgp-helper/DB/OSType.pm
cp DB/DbgrXS.pm blib/lib/dbgp-helper/DB/DbgrXS.pm
cp DB/CGI/Util.pm blib/lib/dbgp-helper/DB/CGI/Util.pm
cp DB/DbgrCommon.pm blib/lib/dbgp-helper/DB/DbgrCommon.pm
cp DB/Data/Dump.pm blib/lib/dbgp-helper/DB/Data/Dump.pm
cp perl5db.pl blib/lib/dbgp-helper/perl5db.pl
cp Devel/Debug/DBGp.pm blib/lib/Devel/Debug/DBGp.pm
cp DB/MIME/Base64.pm blib/lib/dbgp-helper/DB/MIME/Base64.pm
cp DB/DbgrProperties.pm blib/lib/dbgp-helper/DB/DbgrProperties.pm
Running Mkbootstrap for Devel::Debug::DBGp ()
chmod 644 perl5db.bs
rm -f blib/arch/auto/dbgp-helper/perl5db/perl5db.so
gcc  -shared -O2 -g -fstack-protector ARRAY(0xfcf6d0)  -o blib/arch/auto/dbgp-helper/perl5db/perl5db.so         \
                \

/bin/sh: -c: line 0: syntax error near unexpected token `('
/bin/sh: -c: line 0: `gcc  -shared -O2 -g -fstack-protector ARRAY(0xfcf6d0)  -o blib/arch/auto/dbgp-helper/perl5db/perl5db.so   \'
make: *** [blib/arch/auto/dbgp-helper/perl5db/perl5db.so] Error 1
  MBARBON/Devel-Debug-DBGp-0.14.tar.gz
  /usr/bin/make -- NOT OK
'YAML' not installed, will not store persistent state
Running make test
  Can't test without successful make
Running make install
  Make had returned bad status, install seems impossible
```

```
$ perl --version

This is perl 5, version 16, subversion 2 (v5.16.2) built for x86_64-linux
$ cat /etc/redhat-release
Red Hat Enterprise Linux Server release 6.7 (Santiago)
```

Please let me know what other information I can provide, haven't debugged many of these kind of install errors in the past.

**Comment by emarcotte** [comment]
Looks like 0.13 doesn't work either, but 0.12 does.

**Comment by mbarbon** [comment]
This should be fixed by 1e2c20d2d40a2a2dbf21e6915f8217c2a52552a8, included in 0.15 (just uploaded to CPAN).

Thanks!
Mattia
